### PR TITLE
Remove waiting loop for the next block, because we are now can get state from previous block

### LINKED ISSUE #413: Update proof.go to get proof in current block

### PR DESCRIPTION
**Pull request by prin-r**
Fixes #413

### CODE REVIEW

**Review by sorawit — CHANGES_REQUESTED** [review]
This is great!

**Review by sorawit — DISMISSED** [review]

File: chain/client/proof.go [hunk 1]
```diff
@@ -417,6 +416,14 @@ func GetProofHandlerFn(cliCtx context.CLIContext) http.HandlerFunc {
 			return
 		}
 
+		rc, err := cliCtx.Client.Commit(nil)
```
Comment (taobun) [inline chain/client/proof.go:419]: Unused err?

**Review by taobun — CHANGES_REQUESTED** [review]

File: chain/client/proof.go (hunk 1 above)
Comment (sorawit) [reply chain/client/proof.go:419]: Good catch!

Comment (prin-r) [reply chain/client/proof.go:419]: done

**Review by taobun — APPROVED** [review]
lgtm

**Review by sorawit — APPROVED** [review]

**Pull request merged**